Benchmarks/Command.lean

```diff
@@ -33,22 +50,144 @@ elab "benchmark" id:ident declSig:optDeclSig val:declVal : command => do
     {avg}s
   geomean over {n} runs:
     {geomean}s
-
-  indidividual runtimes:
-    {runTimes}
 "
 
 /-- The default `maxHeartbeats` setting.
 
 NOTE: even if the actual default value changes at some point in the future,
 this value should *NOT* be updated, to ensure the percentages we've reported
 in previous versions remain comparable. -/
-def defaultMaxHeartbeats : Nat := 200000
+private def defaultMaxHeartbeats : Nat := 200000
+
+private def percentOfDefaultMaxHeartbeats (heartbeats : Nat) : Nat :=
+  heartbeats / (defaultMaxHeartbeats * 10)
```

Comment:
I must admit, I don't understand the maths here: Shouldn't it return a `Float`, and be `100.0 * (heartbeats / defaultMaxHeartbeats)`

Reply:
Ah, you'd think so, but there is a difference of a factor 1000 in the heartbeats as measured by Lean, and the heartbeats as reported to users. So, the math here is `(heartbeats / (defaultMaxHeartbeats * 1000) * 100`, which cancels out to what I wrote there (please do check my maths).

I guess a comment explaining this is warrented

Reply:
Also, why it doesn't return a float: it could, but we're not super interested in stuff that takes less than 1 percent anyway.